uwblueprint/feeding-canadian-kids
# 29 Auth Login Page UI

## Zenhub ticket link

[29 Auth Login UI](https://app.zenhub.com/workspaces/fck-627480c10d8974001872b0ef/issues/gh/uwblueprint/feeding-canadian-kids/29)



## Implementation description
* Implemented Figma design for desktop and mobile view for log in component



## Steps to test
1. Ensure that front end changes are responsive on all screen sizes
2. Ensure that front end changes match Figma designs 



## What should reviewers focus on?
* Check to ensure that the component is responsive on all screen sizes


## Checklist
